bot.py

```diff
@@ -66,24 +66,25 @@ def send_to_bot(message, increment=False):
         m = u.bot.sendMessage(chat_id=config['telegram']['chat_id'], text=message, parse_mode=ParseMode.MARKDOWN, disable_notification=True)
         lastmsg = (message, m.message_id, 2, time.time())
 
-_someone_waiting_outside = False
+_announce_next_open = False
 
 def on_message(mosq, obj, msg):
-    global _someone_waiting_outside
+    global _announce_next_open
     if msg.topic == 'door/outer/opened/username':
-        if(msg.payload == 'MANUAL OVERRIDE KEY')
-            _someone_waiting_outside = True
-        send_to_bot("*%s* opened the outer door." % msg.payload.decode('utf-8'), increment = (msg.payload != b'MANUAL OVERRIDE KEY'))
+        send_to_bot("*%s* opened the outer door." % msg.payload.decode('utf-8'), increment = True)
     elif msg.topic == 'door/outer/doorbell':
         send_to_bot("%s" % random.choice(['Doorbell'] * 20 + ['llebrooD']), increment = True)
-        _someone_waiting_outside = True
+        _announce_next_open = True
     elif msg.topic == 'door/outer/invalidcard':
-        end_to_bot("Unknown card at door", increment = True)
```

Comment:
What was "end_to_bot"? Could that have been a thing that caused it to fall over?

Reply:
Bug I introduced a month ago :( sorry 

I wouldn't be surprised if that was causing the crashes, good spot

https://github.com/HACManchester/mqtt2telegram/commit/a46040797adf99e8102659a340db0a36b3ca120b#diff-f289368d1ae6d2d58bec9f799832d932R78